iXBRLViewerPlugin/viewer/src/js/viewer.js

```diff
@@ -152,7 +152,7 @@ Viewer.prototype._updateLink = function(n) {
                 e.preventDefault(); 
             });
         }
-        else {
+        else if (file) {
```

Comment:
In theory, this function should do true URL resolution (e.g., `new URL(url, current)`) before looking up the filename.  In practice, same-document reference uses empty string (so, `#id`), so we just check for empty `file` here.  In any case, let the browser do the navigation since the document is already loaded.  I'm not sure if `showElement` should be used (@paulwarren-wk?), but this at least restores the previous behavior.

Reply:
I don't think `showElement` is needed, as we can just let the browser do its thing.